### PR TITLE
[Modular] ACTUALLY Restores some Languages removed by our ball & chain.

### PR DESCRIPTION
**Pull request by theselfish**
## About The Pull Request

Title.
Pirates/skeletons have gotten their themed language back.
image

Nekomantic has been restored to Felinids because the removal reason was "gEnEmOdDeRs iN SkYrAt ""lOrE"" " 
image

As a result, the Japanese language has returned to simply being a choosable language. If you wish to use that instead, it's in the selectable thing. I might switch this around if requested. 

## Why It's Good For The Game

This is essentially a reversion back to TG. 
Pirates should speak like pirates, the players act like pirates already. If requested/required I might look into making the fun language exclusive to the skeletons.
Neko is a species language. I also dislike the Japanese language as a species language.
I am not a saleswoman, I am not good at pitching things. 

## Changelog

:cl:
add: Neko is now the default language of Felinids again.
add: Pirate has been restored to be Piratespeak.
/:cl:

### CODE REVIEW

**Review by nevimer — APPROVED** [review]

File: modular_zubbers/modules/customization/modules/language/_language_holder.dm [hunk 1]
```diff
@@ -0,0 +1,30 @@
+/datum/language_holder/felinid
+	understood_languages = list(
+		/datum/language/common = list(LANGUAGE_ATOM),
+		/datum/language/nekomimetic = list(LANGUAGE_ATOM)
+	)
+	spoken_languages = list(
+		/datum/language/common = list(LANGUAGE_ATOM),
+		/datum/language/nekomimetic = list(LANGUAGE_ATOM)
+	)
```
Comment (Majkl-J) [inline modular_zubbers/modules/customization/modules/language/_language_holder.dm:9]:
```suggestion
/datum/language_holder/felinid/Initialize()
	. = ..()
	understood_languages |= list(
		/datum/language/common = list(LANGUAGE_ATOM),
		/datum/language/nekomimetic = list(LANGUAGE_ATOM)
	)
	spoken_languages |= list(
		/datum/language/common = list(LANGUAGE_ATOM),
		/datum/language/nekomimetic = list(LANGUAGE_ATOM)
	)
```

Doing something like this. Should make it compatible incase skyrat adds more languages to these.
Do mind this is just atop of my head and might not be entirely right, but it works as an example

**Review by Majkl-J — CHANGES_REQUESTED** [review]

File: modular_zubbers/modules/customization/modules/language/_language_holder.dm [hunk 2]
```diff
@@ -0,0 +1,33 @@
+/datum/language_holder/felinid/New()
+	. = ..()
+	understood_languages = list(
+		/datum/language/common = list(LANGUAGE_ATOM),
+		/datum/language/nekomimetic = list(LANGUAGE_ATOM)
+	)
+	spoken_languages = list(
+		/datum/language/common = list(LANGUAGE_ATOM),
+		/datum/language/nekomimetic = list(LANGUAGE_ATOM)
+	)
```
Comment (Majkl-J) [inline modular_zubbers/modules/customization/modules/language/_language_holder.dm:10]:
```suggestion
	understood_languages |= list(
		/datum/language/common = list(LANGUAGE_ATOM),
		/datum/language/nekomimetic = list(LANGUAGE_ATOM)
	)
	spoken_languages |= list(
		/datum/language/common = list(LANGUAGE_ATOM),
		/datum/language/nekomimetic = list(LANGUAGE_ATOM)
	)
```

File: modular_zubbers/modules/customization/modules/language/_language_holder.dm [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+/datum/language_holder/golem/bone/New()
+	. = ..()
+	understood_languages = list(
+		/datum/language/common = list(LANGUAGE_ATOM),
+		/datum/language/piratespeak = list(LANGUAGE_ATOM)
+	)
+	spoken_languages = list(
+		/datum/language/common = list(LANGUAGE_ATOM),
+		/datum/language/piratespeak = list(LANGUAGE_ATOM)
+	)
```
Comment (Majkl-J) [inline modular_zubbers/modules/customization/modules/language/_language_holder.dm:21]:
```suggestion
/datum/language_holder/golem/bone/New()
	. = ..()
	understood_languages |= list(
		/datum/language/common = list(LANGUAGE_ATOM),
		/datum/language/piratespeak = list(LANGUAGE_ATOM)
	)
	spoken_languages |= list(
		/datum/language/common = list(LANGUAGE_ATOM),
		/datum/language/piratespeak = list(LANGUAGE_ATOM)
	)
```

File: modular_zubbers/modules/customization/modules/language/_language_holder.dm [hunk 4]
```diff
@@ -0,0 +1,33 @@
...
+/datum/language_holder/skeleton/New()
+	. = ..()
+	understood_languages = list(
+		/datum/language/common = list(LANGUAGE_ATOM),
+		/datum/language/piratespeak = list(LANGUAGE_ATOM)
+	)
+	spoken_languages = list(
+		/datum/language/common = list(LANGUAGE_ATOM),
+		/datum/language/piratespeak = list(LANGUAGE_ATOM)
+	)
```
Comment (Majkl-J) [inline modular_zubbers/modules/customization/modules/language/_language_holder.dm:32]:
```suggestion
/datum/language_holder/skeleton/New()
	. = ..()
	understood_languages |= list(
		/datum/language/common = list(LANGUAGE_ATOM),
		/datum/language/piratespeak = list(LANGUAGE_ATOM)
	)
	spoken_languages |= list(
		/datum/language/common = list(LANGUAGE_ATOM),
		/datum/language/piratespeak = list(LANGUAGE_ATOM)
	)
```

**Review by Majkl-J — CHANGES_REQUESTED** [review]

**Pull request merged**